docs/content/parent-child-1-ui-init.wsd

```diff
@@ -0,0 +1,24 @@
+@startuml "parent-child-1-ui-init"
+' © Sebastian Porto 2016, 
+' licensed under a Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License.
+autonumber "<b>(#)"
+participant User #white
+participant program #white
+participant Main.initialModel as Mim #white
+participant Main.view as MV #white
+participant Main.update as MU #white
+participant Widget.initialModel as Wim #white
+participant Widget.view as WV #white
+participant Widget.update as WU #white
```

Comment:
Maybe no need to mention the update, since it's not invoked in this diagram?

Reply:
If I'll remove unused participants (`update` for both `Main` and `Widget` in the first image, `initialModel` in the second), images will have different widths (or one of these will be stretched more than the other). Here's how it looks when getting rid of unused participants:

[UI initialisation](http://www.plantuml.com/plantuml/uml/VP71IiD048RlUOeXFQZWbbR4WqXfwKN1Yb1ipfDaQ0TtPyDcqZwJh-6JcMs9BBHwpF_zyys4dRUAGJjd8MymaAYfTcnhCpKT6nRMFBk2xozOKucjCWgy-w0UxYRJXrlecUMAfAKQEgaf0C8o42hl2PRUEIyjB5G3bvso5_Fc9SOKAaPhrZiCjB3yIN0_di2BA0N1M4GBhoVnECDElNIkf03vKpcx7jta6LRg0toqVJOwx5WfQp0eLzoWA3J1RmEwIsY5BEFuNunsvMkoW2si-E_kdkbmx6mkLGgkjwHdmeBTF-r1MVmgi-5YCul7F48gpFeaWk8C514C8uUZF9demKdRGuSi9vHksa1A-i-u9OQfgm_CK7vM5tkfUqWJRtoM8Sxc97Ndx0y0)
[user interaction](http://www.plantuml.com/plantuml/uml/VP3DQi9058NtynGNNLJ1Y2sb2ocYkAdGb88ahcyo5nsSdp1pey_Kr-YJDJzDLB3TXNE-UywPB0EZvz9eyG3VNx2b30ChjF3fF3jucZs-JA3ojCh91f9GMaaU4DQUaDM9OEsCSJR0Yjchh6JbRFJXR2sJpnNgQ7j0JokjZWJFqnbiB9EtM8EeuRqDdWeisTdIPEHXy9ejHiFnG63EpaCIAcruFYWcKQ1dbQi2BKFXtTwZkMV5gEpqfEWC621E_qJAGY9J0oNte9sIU-8-QPV-0rsoTdsMQBP7Y_fo3ckjygEeFYiXJkUmiRad32JYMkhUCuTHcnY7VO-CnQ_Rt2Rt6J2eREmaQH4dJMi2bnOujnUDNpbHbzGEbzV-179d35evRfAkn_LuNzR3joEh5ryFIllVSBEZaZluZKtBNUewTKbMbaR_0000)

I think as these images are shown one above the other, that'll look better if we just leave "participants" lined up in both schemas.